test/kml.test.js

```diff
@@ -14,6 +14,24 @@ test('tokml', function(t) {
         }
         t.equal(expected, output(name + '.kml'), name);
     }
+    
+    function testColor(t, inputColor, inputOpacity, expected) {
+        var featureCollection = file('lineString.geojson');
```

Comment:
this should be `linestring` rather than `lineString` - it fails on Travis and probably succeeds on your machine because your filesystem is case-insensitive.